project.clj

```diff
@@ -0,0 +1,9 @@
+(defproject tic-tac-toe "0.1.0-SNAPSHOT"
+  :description "FIXME: write description"
```

Comment:
You should fill this out

Reply:
Replaced with this:

(defproject tic-tac-toe "0.1.0-SNAPSHOT"
  :description "Tic-Tac-Toe vs Unbeatable Computer"
  :url "https://github.com/jerminatorhits/clojure-tic-tac-toe/"